app/helpers/vita_min_form_builder.rb

```diff
@@ -368,10 +368,19 @@ def vita_min_date_text_fields(method, label_text, help_text: nil, options: {}, o
   def submit(value, options = {})
     options[:data] ||= {}
     options[:data][:disable_with] = value
-    super(value, **options)
+    current_intake_input.html_safe + super(value, **options)
   end
 
   def continue(value = I18n.t("general.continue"))
     submit(value, class: "button button--primary button--wide")
   end
+
+  private
+
+  def current_intake_input
+    # A hidden element containing the identified for the current_intake
+    return "" unless @object.respond_to?(:intake) && @object.intake && @object.intake.id
+    @template.hidden_field_tag(:current_intake, @object.intake.to_global_id)
```

Comment:
```suggestion
    @template.hidden_field_tag(:current_intake, @object.intake.to_signed_global_id)
```
Can we sign these since they're exposed in the client ui?